CMakeLists.txt

```diff
@@ -67,6 +67,18 @@ set (SO_VERSION_GETENV 0)
 # needs version above
 include (cmake/ElektraPackaging.cmake)
 
+# ensure KDB_DB_SYSTEM is an absolute path
+if (NOT IS_ABSOLUTE "${KDB_DB_SYSTEM}")
+	set (KDB_DB_SYSTEM "${CMAKE_INSTALL_PREFIX}/${KDB_DB_SYSTEM}")
+endif ()
+
+# process KDB_DB_SPEC so it is available everywhere
+if (IS_ABSOLUTE "${KDB_DB_SPEC}")
+	set (BUILTIN_SPEC_FOLDER "${KDB_DB_SPEC}")
```

Comment:
Do we actually need BUILTIN_SPEC_FOLDER? Why not directly modify KDB_DB_SPEC as done with KDB_DB_SYSTEM?